css/Components/About.scss

```diff
@@ -0,0 +1,154 @@
+@import "../Theme/Breakpoint";
+@import "../Theme/Color";
+@import "../Theme/Spacing";
+
+$About-content-max-width: 982px;
+
+$About-text-max-width: 534px;
+$About-text-first-letter-font-size: 57px;
+$About-text-first-letter-line-height: 52px;
+$About-text-first-letter-spacing-right: 6px;
```

Comment:
For what I checked in the mocks, this text style will be used in other sections, would you consider extracting it into a component?

For reference, I'm talking about these text blocks:

![screen shot 2017-03-09 at 17 25 12](https://cloud.githubusercontent.com/assets/4236592/23762278/757c9c00-04ed-11e7-8cda-f97fcd1f008b.png)
![screen shot 2017-03-09 at 17 25 17](https://cloud.githubusercontent.com/assets/4236592/23762279/757ecec6-04ed-11e7-917e-bb62e954950c.png)